SITCOMTN-086.tex

```diff
@@ -46,24 +62,823 @@
+have to be taken to ensure that outliers from the spectroscopic fits do not
+contaminate the model used in the broadband photometry.  We do not plan to use
+satellite measurements of atmospheric constituents as priors as the resolution
+is low enough to be contaminated by observations of neighboring valleys rather
+than the mountain peak where the telescope is sited.
+
+\subsection{Integrating Over Sources}
+
+This short section will discuss PSF modeling and aperture photometry and the
+finit size of stars (and galaxies) and the care that needs to be taken with
+fluence or surface density images, following B17.
+
+\subsection{Foregrounds and Backgrounds}
+
+What to do about foregrounds and backgrounds, including fringes?  These needs
+to be written.
+
+\subsection{Global Calibration}
+
+\section{Flat Field System}
+
+
+\section{CBP}
+
+Flatfield system consists of four main parts: a calibration screen, an aspheric reflector optic, a tunable laser and a white light system. 
+
+The calibration screen will be illuminated by either the white light system or a tunable laser. The white light system will be used for daily flats, while the tunable laser will be used to create monochromatic flats. Both illumination systems are mounted on the dome and co-rotate with the calibration screen. The output of each is co-located at the center of the calibration screen and can be switched between. In order to project that light onto the screen, and additional optic is mounted on the back of the camera and pancake wrap. This is located precisely 3 meters from the projector. It is critical that the projector is aligned with the reflector to ensure a flat illumination pattern on the calibration screen. 
+
+\subsection{Calibration Screen}
+The calibration screen is a large reflective surface that can be aligned with the primary mirror and send diffuse light directly to the telescope and camera. It is large enough to illuminate the whole telescope at one time. The surface itself has a diameter of 10.27 meters and an inner diameter of 3.18 meters. Of this area, the section from a diameter of 4.18 - 9.27 meters is coated with a highly reflective material. The outer and inner rings are coated with a very absorptive material. The screen is built of several small panels, with 16 panels making the outer ring and 8 panels making the inner ring. They are secured to a structure such that they are all flat to 3mm across the whole surface. The panels are coated with Labsphere Permaflect 94\% and 5\% respectively. 
+
+The structure on which the panels are mounted can rotate from 0 to -23 degrees relative to horizontal using an actuator. In this way, it can be tilted so that the optical axis of the telescope aligns with teh optical axis of the screen. There will also be 4 retroreflectors mounted around the screen that can be seen by a laser tracker that is mounted on the telescope so that the screen can be easily aligned with the telescope on a daily basis. 
```

Comment:
```suggestion
The structure on which the panels are mounted can rotate from 0 to -23 degrees relative to horizontal using an actuator. In this way, it can be tilted so that the optical axis of the telescope aligns with the optical axis of the screen. There will also be 4 retroreflectors mounted around the screen that can be seen by a laser tracker that is mounted on the telescope so that the screen can be easily aligned with the telescope on a daily basis. 
```